device-types/Cisco/N9K-C9372TX-E.yaml

```diff
@@ -0,0 +1,152 @@
+---
+manufacturer: Cisco
+model: Nexus 9372TX-E
+part_number: N9K-C9372TX-E
+slug: n9k-C9372tx-e
+u_height: 1
+is_full_depth: true
+########################################################################################################
+### airflow: This is configurable based on what Fan and Power Supply modules you have in your device ###
+########################################################################################################
```

Comment:
Please remove the comment from the file, thank you.

Reply:
Removed the comments and updated it with the current airflow configuration we have setup (rear-to-front).